packages/ingest/pdr.js

```diff
@@ -77,10 +77,18 @@ class Discover {
    *   if it does not already exist in S3.
    */
   pdrIsNew(pdr) {
-    return aws.s3ObjectExists({
-      Bucket: this.bucket,
-      Key: path.join(this.stack, this.folder, pdr.name)
-    }).then((exists) => (exists ? false : pdr));
+    const event = {
+      pathParameters: { pdrName: pdr.name }
+    };
+
+    const cb = (err, _) => {
+      if (err) {
+        if (err.name === 'RecordDoesNotExist') return pdr;
+        throw err;
+      }
+      return false;
+    };
+    return getPdr(event, cb);
```

Comment:
I think what I would do here is extract the functionality from `pdrs.get()` in `api/endpoints/pdrs` to a function that's easier to use. 

Something like: 
```
function getPdr(pdrName) {
  const p = new models.Pdr();

  return p.get({ pdrName });
}
```

Then you could have the endpoint call that, expose that as the public function, and use that here without having to do the weird callback stuff.

Reply:
I'm with you, here. The only complication that arises with that implementation is `models.Pdr();` [constructor](https://github.com/cumulus-nasa/cumulus/blob/master/packages/api/models/pdrs.js#L11) requires a `process.env.PdrsTable` environment variable. Definitely becomes a breaking change.

Reply:
Now that I say that... I'm pretty sure that just calling getPdr requires that the environment variable exists.